src/css/components/panel.scss

```diff
@@ -252,3 +230,14 @@ h4 + .panel {
   margin: (-$grid-3) (-$grid-3) $grid-3 (-$grid-3);
   padding: $grid-3 $grid-3 $grid-4;
 }
+
+// a couple random panel styles
+
+.route-link {
```

Comment:
Similar comment to `card`, I'd prefer to keep only things related to `panel` in here so everything is easier to find and it keeps the current organization consistent. Is there a reason it should be in here though?